### PR TITLE
Flashes Knockdown instead of Paralyzing

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Flashes knock people to the floor (so a slowdown) instead of full stun

## Why It's Good For The Game

Personally, I think some other moves may be better, but for the moment flashes need to not paralyze. Better to be bad and enable the use of every other weapon than good and be "the meta"

baseball bats should get this treatment eventually too

## Changelog
:cl:
balance: flashes knockdown instead of paralyze
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
<details>
  <summary>How i'm feelin' making this right now</summary>

  image
  image

</details>

File: code/modules/assembly/flash.dm [hunk 1]
```diff
@@ -126,7 +126,7 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+			M.Knockdown(rand(80,120))
```
Comment (Fikou) [inline code/modules/assembly/flash.dm:129]: can you remove funny randomness

Comment (tralezab) [reply code/modules/assembly/flash.dm:129]: I'm not entirely against it, like to hear what others say.

**Comment by Fhaxaris** [comment]
based

**Comment by necromanceranne** [comment]
86041410-74289280-b9fa-11ea-99aa-8fbc10e010fb

**Comment by wesoda25** [comment]
A lazy solution but preferable to stuns

File: code/modules/assembly/flash.dm (hunk 1 above)
Comment (wesoda25) [reply code/modules/assembly/flash.dm:129]: I think an important aspect of the flash is its unreliability, ranging from its unpredictable burnout to its semi random effects. I don’t know if it would actually matter if you changed this, just make sure you’re consciously doing so.

**Comment by Pugie** [comment]
Still enables stuns better than the stun baton lol

**Comment by tralezab** [comment]
knockdown just slows people

**Comment by evilcyborgmrt** [comment]
Long time coming. Really though they should just temporarily blind and nothing else. Why would a bright light knock someone on their ass anyway?

**Comment by tralezab** [comment]
same reason it invokes full body paralysis: ???

**Comment by ATH1909** [comment]
but borgs

**Comment by tralezab** [comment]
what about em this is the effect on carbons no

**Comment by wesoda25** [comment]
Prophecy
9Ux8yxMkvm
chrome_liThbFmn9N

**Comment by PKPenguin321** [comment]
anger

**Comment by Acer202** [comment]
Hard stuns ruin so many potential fights. Good riddance.

**Comment by ATH1909** [comment]
> what about em this is the effect on carbons no

they can't capitalize on knockdowns like humans can; they don't get shovestuns or the like (and standard cyborgs can't cuff someone who's merely knocked down (provided that they aren't braindead)). furthermore, the humans you knock down can still just whip a flash out of their backpack with a hotkey and end the fight in a single click, and you can't really stop them from doing that, because borgs have been specifically designed to be melee-focused (and the few ranged options that you do have (siphoning the air out of a room, using a mining cyborg's PKA, shuttercrushing idiots who practically stand on a giant red target) usually involve H A R M).

in short, this PR will absolutely fuck over the combat capabilities of borgs who don't drag a secbot around behind them 24/7.

**Comment by wesoda25** [comment]
Ngl I think its high time we rethink borgs

**Comment by ATH1909** [comment]
also, this is gonna REALLY dick over malf AIs who want to use the forceborger, especially if my stun arm nerf/change PR goes through (speaking of which, I need to get back to that...).

**Comment by tralezab** [comment]
@wesoda25 i completely agree but i'm not looking to break silicon gameplay so maybe i'll just like make a subtype borgs use that is paralyze just to keep their weird entirely stun combat

**Comment by Pugie** [comment]
flash still better than stunbaton

**Comment by tralezab** [comment]
I heard you so do you have a better idea? Why not just post in the thread we're discussing it on right now? Arrr, we're trying to figure this out!!!

**Comment by anconfuzedrock** [comment]
ok so its just a worse baton then
why not make it dizzy or something so it has an actual purpose

**Comment by carshalash** [comment]
Yeah no, this just makes teleport in front of core kills even easier.

**Comment by Nistishere** [comment]
still better than stunbaton + silicon nerf

**Comment by Pugie** [comment]
Giving it knockdown is dumb because instant knockdown is better than what stun baton offers as it's basically an instant shove, which you can then get a 4 second stun from. In any 1 on 1 combat that stun will end the fight then and there 4 seconds might not seem like much but you can be extremely malleable with that stun. It is a nerf to a trash item and I suppose I shouldn't look a gift horse in the mouth, but the only difference functionally is the stun is now 10 seconds shorter which isn't much.

**Comment by Dorsisdwarf** [comment]
arent borgs about to lose electroprods because "Just use the flash to stun people lol"

**Comment by Firecage** [comment]
@Dorsisdwarf Well, to be perfectly honest. In my opinion. One can either keep the electropods or think of something else in regards to them, since they are antag only most of the time. Flashes on the other hand, are everywhere, and more can easily be printed. Them stunning people are, frankly, silly.

**Comment by necromanceranne** [comment]
Pugie basically hit the nail on the head. We still have a whole host of problems that are deep rooted throughout combat as a whole, and shove stuns is one of those big ones, Knockdowns are going to lead to a pretty hefty stun, so regardless of when or how, knockdowns are usually just stuns but on a two press rather than one press. This is also what I was saying a lot with stun batons and telebatons, but fuck me if I'll get listened too. 

Last time someone tried to remove shove stuns, I think oranges instaclosed it for some reason without elaboration, so trying to figure out an alternative to that is a fucking mess. I suspect it's to do with not being able to 'remove people from departments', but I feel stuns are also not the solution there, but any good change for nonlethal takedowns should be a more game wide change, like creating more nonlethal weaponry or converting some weapons to nonlethal. 

Shoves enable a naked with perfect timing to wipe a room, assuming it is well shaped, and most rooms are. The 3x3 is exactly what you want. Flashes can still be stuns, and shoves can put you into an effective stunlock with grab spam.

**Comment by Acer202** [comment]
Keep in mind that the autoborger PR makes a very important change to consider: the borger works with corpses. What does that mean if you get stunned by a borg now? It means they're going to kill you on the spot to make their life easier. More stun-and-done gameplay if the borg's flash still instantly paralyzes. Hard stuns make for very non-thrilling gameplay, and I think they have no business existing for borgs or humans in a one-click fashion. Having such a simple one-click "I win" button to use circumvents a thrilling chase, a struggle. You know, adrenaline-filled gameplay. Much better than rolling your eyes because there's literally nothing you can do but wait for them to welder you to death since you didn't powergame flash protection.

**Comment by IndieanaJones** [comment]
> Keep in mind that the autoborger PR makes a very important change to consider: the borger works with corpses. What does that mean if you get stunned by a borg now? It means they're going to kill you on the spot to make their life easier. More stun-and-done gameplay if the borg's flash still instantly paralyzes. Hard stuns make for very non-thrilling gameplay, and I think they have no business existing for borgs or humans in a one-click fashion. Having such a simple one-click "I win" button to use circumvents a thrilling chase, a struggle. You know, adrenaline-filled gameplay. Much better than rolling your eyes because there's literally nothing you can do but wait for them to welder you to death since you didn't powergame flash protection.

Problem with borgs is a two-way street.  They can stun people and kill them with impunity, but also get stunned and murdered.  To up this, cyborgs currently have a stun arm to let them stun people with flash protection, but since that's getting removed, the borg is now always in the losing situation.

**Comment by tralezab** [comment]
So what i'm getting from this (and again i told you to discuss in the thread 😡) is that knockdown isn't enough EXCEPT for borgs which should actually keep the paralyze?

**Comment by tralezab** [comment]
@necromanceranne this isn't related but honestly if you couldn't chain stuns (like add onto them) wall shoving would actually be a pretty good mechanic

**Comment by Acer202** [comment]
> Problem with borgs is a two-way street. They can stun people and kill them with impunity, but also get stunned and murdered. To up this, cyborgs currently have a stun arm to let them stun people with flash protection, but since that's getting removed, the borg is now always in the losing situation.

Aye, what I think is that the flash effect on a borg should also result in something other than a hard stun. Maybe a period of blindness, confusion/random movement for the borg. Anything other than instant hard stuns for either side. If the neither side has such stuns, I think things could get a lot more interesting, struggle-wise. Borgs could still use carried bodies as projectiles for a ranged stun, if they're really out to hard-stun still.

**Comment by wesoda25** [comment]
I proposed a while ago that borgs flashing borgs would result in a temporary slowdown, module disablement, and blind effect, but noooo borg mains wanted to keep their shitty system of fighting

**Comment by carshalash** [comment]
This kinda makes autoflashers completely useless as well.

**Comment by Pugie** [comment]
> @necromanceranne this isn't related but honestly if you couldn't chain stuns (like add onto them) wall shoving would actually be a pretty good mechanic

You can't chain wall stuns

**Comment by tralezab** [comment]
yeah i tested in game after that post you're totally right, dunno what all this hullabaloo complaining about shoving is then

**Comment by tralezab** [comment]
I'm just not interested in balance while I have so many cool projects this is just like the huge ugly wart in my closet

**Comment by wesoda25** [comment]
Sad!
